### PR TITLE
Update Aruba test suite to contain the latest changes

### PR DESCRIPTION
**Pull request by honzajavorek**
The [dredd-hooks-template](https://github.com/apiaryio/dredd-hooks-template) was updated with some minor changes. This PR is an attempt to update the local copy to contain those changes.

The PR doesn't pass and unfortunately I won't have much time soon to investigate why, so I'll just keep it here in progress for a while... 😕 

**Related to:** https://github.com/apiaryio/dredd-hooks-template/pull/9, https://github.com/apiaryio/dredd-hooks-template/pull/10, https://github.com/apiaryio/dredd-hooks-template/pull/12, https://github.com/apiaryio/dredd-hooks-template/pull/13, https://github.com/apiaryio/dredd-hooks-template/pull/14

**Should be coordinated with:** https://github.com/apiaryio/dredd-hooks-template/pull/15 and @gonzalo-bulnes' changes in [dredd-hooks-ruby](https://github.com/apiaryio/dredd-hooks-ruby).

### CODE REVIEW

**Comment by w-vi** [comment]
@honzajavorek Do you mind if I look into it?

**Comment by honzajavorek** [comment]
@w-vi Sure, you're welcome to help! I updated the description.

File: features/multiple_hookfiles.feature [hunk 1]
```diff
@@ -47,7 +47,7 @@ Feature: Multiple hookfiles with a glob
           print("It's me, File3")
 
       """
-    When I run `dredd ./apiary.apib http://localhost:4567 --server "ruby server.rb" --language "dredd-hooks-python" --hookfiles ./hookfile1.py --hookfiles ./hookfile2.py --hookfiles ./hookfile_*.py`
+    When I run `dredd ./apiary.apib http://localhost:4567 --server="ruby server.rb" --language="dredd-hooks-python" --hookfiles=./hookfile1.py --hookfiles=./hookfile2.v --hookfiles=./hookfile_*.py`
```
Comment (gonzalo-bulnes) [inline features/multiple_hookfiles.feature]: Hi @honzajavorek, this looks like a misspelling: `--hookfiles=./hookfile2.v` should be `--hookfiles=./hookfile2.py`, which I think explains the [first error](https://travis-ci.org/apiaryio/dredd-hooks-python/jobs/194513087#L613). (But probably not the second.)

**Comment by gonzalo-bulnes** [comment]
@honzajavorek @w-vi The [second error](https://travis-ci.org/apiaryio/dredd-hooks-python/jobs/194513087#L672) in the last build is a timeout. I'll update https://github.com/apiaryio/dredd-hooks-template/issues/16 with details because it does affect everyone.

File: features/multiple_hookfiles.feature (hunk 1 above)
Comment (honzajavorek) [inline features/multiple_hookfiles.feature]: Indeed! Thanks

**Comment by honzajavorek** [comment]
The "here" link in your comment leads to the same comment itself :) I think you meant https://github.com/apiaryio/dredd-hooks-template/issues/16.

**Comment by gonzalo-bulnes** [comment]
Oops, yes!

**Comment by honzajavorek** [comment]
Superseded by https://github.com/apiaryio/dredd-hooks-python/pull/49

**Pull request closed without merging**